### ISSUE TITLE
Changing Error Message in v1.4

### ISSUE THREAD

**Issue by lennoxtr** [nus-cs2103-AY2223S2/forum#364]
Is it possible for us to change the error message for v1.4 if what is shown to the user is wrong (such as an error message about exporting file when user is importing)

Thank you!

**Comment by damithc** [comment]
@lennoxtr Yes, if the error message is incorrect, that's a bug you are allowed to fix. For more info, see Q6 [here](https://nus-cs2103-ay2223s2.github.io/website/schedule/week12/project.html#:~:text=FAQs%20on%20what%20is%20allowed%20during%20the%20feature%20freeze%3A).
